src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1036,10 +1095,10 @@ type Place {
     bikeRentalStation: BikeRentalStation
 
     """The bike parking related to the place"""
-    bikePark: BikePark
+    bikePark: VehicleParking
 
     """The car parking related to the place"""
-    carPark: CarPark
+    carPark: VehicleParking
```

Comment:
In order to keep backwards compatibility with digitransit we need to keep this a CarPark. Can you please change that?